chat-plugins/eShop.js

```diff
@@ -65,120 +65,88 @@ try {
 }
 
 exports.commands = {
-	//shop: 'eshop', //Uncomment this if you want this to be able to be used using the /shop command
 	eshop: {
 		add: function (target, room, user, connection, cmd, message) {
 			if (!this.can('roomowner')) return false;
-			if (!allowThisShop) return this.errorReply('This shop is closed');
 			if (SG.eShop.closed) return this.sendReply('An error closed the shop.');
 			target = target.split(',');
 			if (!target[2]) return this.parse('/eshop help');
 			if (SG.eShop[toId(target[0])]) return this.errorReply(target[0] + ' is already in the shop.');
 			if (isNaN(Number(target[2]))) return this.parse('/eshop help');
-			let isSSB = false;
-			if (toId(target[0]) === 'shiny' || toId(target[0]) === 'ffacustomsymbol' || toId(target[0]) === 'customability' || toId(target[0]) === 'customitem' || toId(target[0]) === 'custommove') isSSB = true;
-			SG.eShop[toId(target[0])] = new NewItem(target[0], target[1], target[2], isSSB);
```

Comment:
These are the handlers for SSBFFA purchases. Since this shop is used mostly by other servers, this is the only way shop can interact with SSBFFA. It would be confusing since SSBFFA has buttons that send you to shop to buy things for it, to not have this shop support SSBFFA.